.discourse-compatibility

```diff
@@ -1 +1,2 @@
+3.1.0: 3e7c99de894b03e9161d9481da118fe893991a1d
```

Comment:
We need to cover future 'minor' releases of stable here. So let's use the new `<` syntax:
```suggestion
< 3.2.0.beta1-dev: 3e7c99de894b03e9161d9481da118fe893991a1d
```